interpreters/duktape/Makefile

```diff
@@ -0,0 +1,56 @@
+############################################################################
+# interpreters/duktape/Makefile
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include $(APPDIR)/Make.defs
+
+DUKTAPE_VERSION = 2.5.0
+DUKTAPE_UNPACK  = duktape-$(DUKTAPE_VERSION)
+DUKTAPE_TARBALL = duktape-$(DUKTAPE_VERSION).tar.xz
+DUKTAPE_URL     = https://github.com/svaarala/duktape/releases/ \
+                    download/v$(DUKTAPE_VERSION)/$(DUKTAPE_TARBALL)
+
+CSRCS = $(DUKTAPE_UNPACK)/src-noline/duktape.c
```

Comment:
@no1wudi  @xiaoxiang781216 Don't you need to extend `clean::` to delete the object generated by this file?